install/merger/types/config_map_client.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import (
+	"context"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ConfigMapClient interface {
```

Comment:
Useful trick for interfaces. You could annotate the interface to be able to use go generate:
`//go:generate mockery -name= ConfigMapClient`

Reply:
I did not commit generated classes for Client interfaces. Now I added additional mock for merger's Data.